### PR TITLE
[WebProfilerBundle] Sticky ajax window

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #23125
| License       | MIT
| Doc PR        | -

This toggles the ajax toolbar block being sticky on click. I find it quite useful in heavy ajax apps :)

Not sure the state needs to be persisted in local storage or so, could be done :) however for our app all the navigating happens via react router, hence no real need for us to persist it between requests.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I'm not sure about this because it introduces a different behavior in the toolbar depending on the panel the user clicks.

**Comment by ro0NL** [comment]
Well.. currently it's the only block (by default) that has no click handler (ie. is not a link). So it already behaves different, but in a more frustrating way :)

it's truly convenient :))

**Review by nicolas-grekas — APPROVED** [review]
:+1: as explained by @ro0NL the "click" action should do something useful, here it is :)

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig [hunk 1]
```diff
@@ -295,15 +299,19 @@
...
 
-.sf-toolbar-block:hover {
+.sf-toolbar-block:hover,
+.sf-toolbar-block.hover {
     position: relative;
 }
-.sf-toolbar-block:hover .sf-toolbar-icon {
+.sf-toolbar-block:hover .sf-toolbar-icon,
+.sf-toolbar-block.hover .sf-toolbar-icon
+{
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig]: this curly brace should stay on previous line for consistency with all other rules

**Comment by ro0NL** [comment]
i think we should do this for all blocks that are not linking btw =/ (ie. `.sf-toolbar-block > .sf-toolbar-icon` vs. `.sf-toolbar-block > a >.sf-toolbar-icon`) if that's the standard format.

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request closed without merging**